package.json

```diff
@@ -5,26 +5,36 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "start": "next start"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
     "@heroicons/react": "^2.0.18",
+    "@mui/material": "^5.13.5",
     "@tailwindcss/forms": "^0.5.3",
-    "@types/node": "20.3.1",
+    "@types/node": "16.10.0",
     "@types/react": "18.2.12",
     "@types/react-dom": "18.2.5",
     "autoprefixer": "10.4.14",
+    "dotenv": "^16.3.1",
     "eslint": "8.42.0",
-    "eslint-config-next": "13.4.5",
-    "next": "^13.4.5",
+    "eslint-config-next": "^12.0.4",
+    "express": "^4.18.2",
```

Comment:
Curious what we're using `express` for? My understanding is we don't need it because Next already provides server-side rendering.